README.md

```diff
@@ -21,106 +27,26 @@ Note: The QuorumChain consensus algorithm is not yet supported by this release.
 
 <a href="https://github.com/jpmorganchase/quorum/wiki/Transaction-Processing#private-transaction-process-flow">![Quorum privacy architecture](https://github.com/jpmorganchase/quorum-docs/raw/master/images/QuorumTransactionProcessing.JPG)</a>
```

Comment:
These links are pointing to fork origin. We should also updated these to `https:://github.com/DES-Network/des-quorum/.... `